static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -267,6 +324,30 @@ define([
                     } else {
                         resolve();
                     }
+
+                    if (abVideoAutoplay) {
+                        // Annoyingly we pass the `parentNode` as the video is absolutely positioned.
+                        var parentNode = player.el().parentNode;
+                        var firstAutoplay = true;
+                        var elementInView = ElementViewable(parentNode, parentNode.clientHeight * (3 / 4), function () {
```

Comment:
`1/2` seemed a little too enthusiastic, almost like you started playing before you might want to.